api/python/src/cellxgene_ontology_guide/ontology_parser.py

```diff
@@ -301,6 +301,38 @@ def map_term_descendants(self, term_ids: Iterable[str], include_self: bool = Fal
 
         return descendants_dict
 
+    def get_term_subtree(self, term_id: str) -> Dict[str, List[Dict[str, List[Any]]]]:
+        """
+        Get the ontology subtree, with the given term as the root node. Only includes terms from the same ontology as
```

Comment:
Where are we filtering for terms in the same ontology?

Reply:
the builder does it https://github.com/chanzuckerberg/cellxgene-ontology-guide/blob/main/tools/ontology-builder/src/all_ontology_generator.py#L154

Reply:
It sounds misleading to include that comment here (while true) and also something we are likely to forget to update if we change that in the builder.

Reply:
What do you think about removing that part from the docstring.

Reply:
I think we should include it in the docs though, so users are aware of what output to expect. It would be more misleading to not say anything, have users expect to see a full subtree, and only get a filtered subset

Reply:
and at that point, this seems like the most visible spot to put it. If we update the builder, it should be part of the work to update all docs/docstrings accordingly

Reply:
it should be, but we will probably forget.